### ISSUE TITLE
Wd detects it as coinminer threat 

### ISSUE THREAD

**Issue by Extremal99** [UnamSanctam/SilentCryptoMiner#2072]
Miner gets detected for schost.exe

**Comment by gabjohn3** [comment]
Of course it does, you have to create own crypter, before trying to play with tools u dont understand please go back to study and create a decent crypter buddy  lol

**Comment by Extremal99** [comment]
Not talking to you ive already developed.the crypter

**Comment by shainodev** [comment]
If you've already developed the crypter and "Wd detects it as coinminer threat" then you obviously haven't done a good job. @gabjohn3 told you to create a "decent" crypter and not a bad one.

**Comment by kaze03** [comment]
shut up all you know-it-alls, he just confirmed with the miner developer that the anti-virus is detecting this threat so that it can be fixed in the future.
and this is not about personal consumption.
i know this man and he has more clients than you can imagine, so it would be good for everyone to mind their own business.

**Comment by gabjohn3** [comment]
> shut up all you know-it-alls, he just confirmed with the miner developer that the anti-virus is detecting this threat so that it can be fixed in the future. and this is not about personal consumption. i know this man and he has more clients than you can imagine, so it would be good for everyone to mind their own business.

i just told him to develop a decent crypter even if he ask i could help to share a better crypter for free but ppl these days are so mean so nevermind :D i will be in my own business good luck xd

**Comment by kaze03** [comment]
bro lmao, even private stubs getting detected wym free crypter

> > shut up all you know-it-alls, he just confirmed with the miner developer that the anti-virus is detecting this threat so that it can be fixed in the future. and this is not about personal consumption. i know this man and he has more clients than you can imagine, so it would be good for everyone to mind their own business.
> 
> i just told him to develop a decent crypter even if he ask i could help to share a better crypter for free but ppl these days are so mean so nevermind :D i will be in my own business good luck xd

**Comment by gabjohn3** [comment]
> bro lmao, even private stubs getting detected wym free crypter
> 
> > > shut up all you know-it-alls, he just confirmed with the miner developer that the anti-virus is detecting this threat so that it can be fixed in the future. and this is not about personal consumption. i know this man and he has more clients than you can imagine, so it would be good for everyone to mind their own business.
> > 
> > 
> > i just told him to develop a decent crypter even if he ask i could help to share a better crypter for free but ppl these days are so mean so nevermind :D i will be in my own business good luck xd

im not talking about crypters with shared stub lol, the "priivate" crypters  coded in c# haha that are in the market  are useless, one crypter that i have in production it have like 3 months fud without issues haha  and i think will be fud at least another 3 months

**Comment by kaze03** [comment]
so weird, alr gl -_-

**Comment by UnamSanctam** [comment]
Yes it seems like Windows has added further scans on programs like that, so I will have to modify the process injection in some way. With explorer.exe it's still undetected.
